frida_tools/tracer.py

```diff
@@ -918,6 +926,29 @@ def _sync_handlers(self, change_id):
                 self._notify_update(function, new_handler, handler_file)
 
 
+class OutputFile(object):
+    def __init__(self, filename, truncate=False):
+        self.filename = filename
+        self._truncate = truncate
+        self._fd = None
+
+    def __enter__(self):
+        if self.filename:
+            self._fd = codecs.open(self.filename, 'ab', 'utf-8')
+            if self._truncate:
+                self._fd.truncate(0)
```

Comment:
As we're not using the truncate option we should remove it.

Reply:
its being used already

Reply:
but i have refactored the code to not need it

Reply:
👍